web/cobrands/fixmystreet/fixmystreet.js

```diff
@@ -1283,14 +1293,23 @@ $.extend(fixmystreet.set_up, {
 
   reporting_required_phone_email: function() {
     var fem = $('#form_email');
+    var fem_optional = $('#js-optional-email-update-method-triggered');
     var fph = $('#form_phone');
+    var fph_optional = $('#js-optional-phone-update-method-triggered');
+
     $('#update_method_email').on('change', function() {
       fem.prop('required', true);
-      fph.prop('required', false);
+      fem_optional.addClass('hidden-js');
+      if (!fixmystreet.selected_category_data.phone_required) {
+          fph.prop('required', false);
+          fph_optional.removeClass('hidden-js');
+      }
```

Comment:
Do we need to make this required/hidden if phone_required is set at this point - otherwise it won't be required, I think.

Reply:
Argh, I was convinced I hadn't missed an edge case this time 🙃